modules/apps/ai-creator/ai-creator-openai-web/src/main/resources/META-INF/resources/ai_creator_modal/AICreatorImageModal.tsx

```diff
@@ -60,6 +62,19 @@ export default function AICreatorImageModal({
 		}
 	};
 
+	const onSelectedChange = (imageURL: string) => {
+		const newSelectedImages = [...selectedImages] || [];
```

Comment:
You are right, `|| []` and even the `?.` are not necessary. The `selectedImages` array is initialized as `[]` so it will be never `null`. Thanks for catching this.